Gems/Atom/RPI/Code/External/MaskedOcclusionCulling/CompilerSpecific.inl

```diff
@@ -82,9 +81,7 @@
 		__cpuid_count(function, subfunction, cpuinfo[0], cpuinfo[1], cpuinfo[2], cpuinfo[3]);
 	}
 
-#ifdef LINUX
-#undef _xgetbv
-#endif
+#if !defined(_xgetbv)
```

Comment:
I'm pretty sure this only works if `_xgetbv` is a preprocessor definition (macro)
If a library defines it as an actual normal function without hiding it behind a macro, then this will run afoul of having 2 functions of the same shape defined.

This is probably still okay, as long as every compiler or platform we use, always uses a preprocessor macro to define `_xgetbv` if its available.

Reply:
Just based on this PR it looks like `PAL_TRAIT_BUILD_ATOM_RPI_MASKED_OCCLUSION_CULLING_SUPPORTED` having that function already. The question is if `_xgetbv` is the only reason for `PAL_TRAIT_BUILD_ATOM_RPI_MASKED_OCCLUSION_CULLING_SUPPORTED` and if so, it could just be renamed? Might also be useful in case other features need `_xgetbv`?